platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/collections/AbstractLongList.java

```diff
@@ -126,6 +128,9 @@ public abstract class AbstractLongList<C> implements LongList {
      */
     protected final long reservedBufferLength;
 
+    /** Platform configuration */
+    protected Configuration configuration;
```

Comment:
I don't see any added lines of code that use this field. Is it needed?

Reply:
Used in `LongListDisk` which inherits from this abstract class and overrides the `onEmptyOrAbsentSourceFile` and `readBodyFromFileChannelOnInit` methods in order to construct `fileSystemManager` for temp files.